scanpipe/pipelines/match_to_purldb.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# http://nexb.com and https://github.com/nexB/scancode.io
+# The ScanCode.io software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode.io is provided as-is without warranties.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# Data Generated with ScanCode.io is provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+# ScanCode.io should be considered or used as legal advice. Consult an Attorney
+# for any legal advice.
+#
+# ScanCode.io is a free software code scanning tool from nexB Inc. and others.
+# Visit https://github.com/nexB/scancode.io for support and download.
+
+from scanpipe.pipelines import Pipeline
+from scanpipe.pipes import purldb
+
+
+class MatchToPurlDB(Pipeline):
+    """Check CodebaseResources of a Project against PurlDB for Package matches."""
+
+    is_addon = True
+
+    @classmethod
+    def steps(cls):
+        return (
```

Comment:
Since this pipeline depends on the proper setup and availability of PurlDB, what about an explicit check as the initial step, similar to https://github.com/nexB/scancode.io/blob/main/scanpipe/pipelines/find_vulnerabilities.py#L44